CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the ConfigCat SDK for Elixir
+
+ConfigCat SDK is an open source project. Feedback and contribution are welcome. Contributions are made to this repo via Issues and Pull Requests.
+
+## Submitting bug reports and feature requests
+
+The ConfigCat SDK team monitors the [issue tracker](https://github.com/configcat/elixir-sdk/issues) in the SDK repository. Bug reports and feature requests specific to this SDK should be filed in this issue tracker. The team will respond to all newly filed issues.
+
+## Submitting pull requests
+
+We encourage pull requests and other contributions from the community. 
+- Before submitting pull requests, ensure that all temporary or unintended code is removed.
+- Be accompanied by a complete Pull Request template (loaded automatically when a PR is created).
+- Add unit or integration tests for fixed or changed functionality.
+
+When you submit a pull request or otherwise seek to include your change in the repository, you waive all your intellectual property rights, including your copyright and patent claims for the submission.
+
+In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr)
+
+1. Fork the repository to your own Github account
+2. Clone the project to your machine
+3. Create a branch locally with a succinct but descriptive name
+4. Commit changes to the branch
+5. Following any formatting and testing guidelines specific to this repo
+6. Push changes to your fork
+7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
+
+## Build instructions
+
+Install [Elixir](https://elixir-lang.org) development environment.
+
+To install dependencies:
+
+```bash
+mix local.rebar --force
+mix local.hex --force
+mix deps.get
+```
+
+## Running tests
+
+```bash
+mix coveralls.json
```

Comment:
For running tests locally, people might prefer to use `mix test` or `mix test.watch` rather than `mix coveralls.json`.

Reply:
thanks I've updated the PR with `mix test`.